src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/basic-alerting-concepts.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Alerting concepts and terms
+tags:
+  - Alerts and applied intelligence
+  - Applied intelligence
+  - Incident intelligence
+metaDescription: 'Some basic terms and concepts for New Relic alerting and applied intelligence.'
+redirects:
+  - /docs/enable-new-relic-ai-0
+  - /docs/enable-new-relic-ai
+  - /docs/new-relic-one/use-new-relic-one/new-relic-ai/get-started-incident-intelligence
+  - /docs/alerts-applied-intelligence/incident-intelligence/get-started-incident-intelligence
+  - /docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/eu-us-datacenters-incident-intelligence
+---
+
+import alertConceptsDiagram from 'images/alerting-concepts-diagram.png'
+
+To make the best use of New Relic alerting and applied intelligence features, it will help you to understand some basic terms and concepts. In the following explanation, the bolded words are important New Relic terms.
+
+<img
+  title="A diagram showing some basic concepts and terms for New Relic alerting and applied intelligence"
+  alt="A diagram showing some basic concepts and terms for New Relic alerting and applied intelligence"
+  src={nraiDecisionsIndex}
+/>
+
+You can set up alerts for your telemetry data by configuring alert **conditions**, which define what will constitute a violation and trigger an **incident**. For example, you might want to create an alert condition that opens an incident if a host's CPU goes above 80%. Multiple incidents can be aggregated into an **issue**. 
+
+Conditions live inside an alert **policy**. The policy contains higher-level settings that govern all the conditions in that policy. One setting is **issue creation preference**, which governs how incidents gets turned into issues. For example, you might want incidents that happen very close together in time to be grouped together in the same issue. We have built-in **decisions**, which are algorithms that automatically group incidents into issues, and you can create your own custom decisions. 
```

Comment:
This is a neat idea, but you can't configure it. I would recommend using an example that is currently configurable (by policy, by condition or "every incident gets its own issue").